tcl/pd_menus.tcl

```diff
@@ -575,6 +575,44 @@ proc ::pd_menus::create_preferences_menu {mymenu} {
         -command {::pd_menus::loadpreferences}
     $mymenu add command -label [_ "Forget All..."] \
         -command {::pd_menus::forgetpreferences}
+		
+	switch -- $::windowingsystem {
+       
+	   "win32" {
+	
+    $mymenu add command -label [_ "Reset .pd files to *this* Pd"] \
+        -command { 
```

Comment:
could you refactor the command into a `proc`?
So it could be re-used  from various places (e.g. if we want to have this option in a separate pref-window)